crates/containerd-shim-wasm/src/sandbox/containerd/client.rs

```diff
@@ -176,6 +181,8 @@ impl Client {
                 }
                 Err(e) => return Err(ShimError::Containerd(e.to_string())),
             };
+
+            // Get initial Stat response
             let response = response_stream
```

Comment:
nit: use `send_message()` for the initial `stat` request.

Reply:
I did try that originally but the process would hang if the receiver passed to the `ReceiverStream` did not have any messages enqueued. I am not sure why this was the case. I can try to debug more

Reply:
That's fine. I do not want to block this PR for it